From: Colin Ian King <colin.king@canonical.com>
Date: Fri, 18 Aug 2017 16:30:00 +0100
Subject: net: hns3: fix a handful of spelling mistakes
Patch-mainline: v4.14-rc1
Git-commit: d7629e748ee4332af91c1c2e2b4d7a92e7d5cde4
References: bsc#1104353 FATE#326415

Trival fix to spelling mistakes:

firware -> firmware
invald -> invalid
mutilcast -> multicast

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/ethernet/hisilicon/hns3/hns3pf/hclge_cmd.c  |    2 +-
 drivers/net/ethernet/hisilicon/hns3/hns3pf/hclge_main.c |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/net/ethernet/hisilicon/hns3/hns3pf/hclge_cmd.c
+++ b/drivers/net/ethernet/hisilicon/hns3/hns3pf/hclge_cmd.c
@@ -334,7 +334,7 @@ int hclge_cmd_init(struct hclge_dev *hde
 	}
 	hdev->fw_version = version;
 
-	dev_info(&hdev->pdev->dev, "The firware version is %08x\n", version);
+	dev_info(&hdev->pdev->dev, "The firmware version is %08x\n", version);
 
 	return 0;
 err_csq:
--- a/drivers/net/ethernet/hisilicon/hns3/hns3pf/hclge_main.c
+++ b/drivers/net/ethernet/hisilicon/hns3/hns3pf/hclge_main.c
@@ -1952,7 +1952,7 @@ int hclge_cfg_mac_speed_dup(struct hclge
 			       HCLGE_CFG_SPEED_S, 5);
 		break;
 	default:
-		dev_err(&hdev->pdev->dev, "invald speed (%d)\n", speed);
+		dev_err(&hdev->pdev->dev, "invalid speed (%d)\n", speed);
 		return -EINVAL;
 	}
 
@@ -3476,7 +3476,7 @@ int hclge_rm_mc_addr_common(struct hclge
 	} else {
 		/* This mac addr do not exist, can't delete it */
 		dev_err(&hdev->pdev->dev,
-			"Rm mutilcast mac addr failed, ret = %d.\n",
+			"Rm multicast mac addr failed, ret = %d.\n",
 			status);
 		return -EIO;
 	}
